src/redis.nim

```diff
@@ -126,7 +126,11 @@ proc managedSend(
   r: Redis | AsyncRedis, data: string
 ): Future[void] {.multisync.} =
   when r is Redis:
-    r.socket.send(data)
+    when defined(windows):
+      if not r.socket.tryRecv(data):
```

Comment:
Why would `tryRecv` + `close` be equivalent to `send`?

Reply:
sorry for that, should be `trySend(`, see the update below